### DISCUSSION TITLE
azure account

### DISCUSSION THREAD

**Discussion by Sudoo69** [github/india#335] [Students]
question might be silly but which account are you guys making? im aware of the module which teaches you how to make a free account but that thing need info of the organization lol idk what should i put there. there is another way to sign up using a student status. i wanted to know what you guys did and if the student subscription had all the tools needed for this internship.

**Comment by TheSuranaverse** [comment]
@Sudoo69 
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together!
A one-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏
